src/main/java/org/hisp/dhis/integration/rapidpro/route/PullRapidProFlowsRouteBuilder.java

```diff
@@ -61,8 +65,13 @@ protected void doConfigure()
         from( "direct:pull" )
             .routeId( "Scan RapidPro Flows" )
             .streamCaching()
+            .process( exchange -> {
+                exchange.setProperty( "flowUuids",
+                    String.join( ",", programStageToFlowMap.getFlowUuids(),
+                        exchange.getContext().resolvePropertyPlaceholders( "{{rapidpro.flow.uuids:}}" ) ) );
```

Comment:
I've previously tried injecting it using `@Value`. It works fine in production, but it causes issues in the test environment, since the bean is initialized before the `rapidpro.flow.uuids` property values is set in the tests. Do you know if there is a simple workaround for this? Looked into creating a custom context initializer, but I feel this might be an overkill for such a seemingly trivial task.

Reply:
I inject it as follows:
``` 
@Value( "${rapidpro.flow.uuids}" )
private String aggregateReportFlowUuids;
```
And set the property in the tests as follows, making sure that all of the properties are set before starting the Camel context:
```
        System.setProperty( "sync.rapidpro.contacts", "true" );
        System.setProperty( "rapidpro.flow.uuids", aggregateReportFlowUuid );
        AdviceWith.adviceWith( camelContext, "Transmit Report", r -> r.weaveAddLast().to( "mock:spy" ) );
        MockEndpoint spyEndpoint = camelContext.getEndpoint( "mock:spy", MockEndpoint.class );
        spyEndpoint.setExpectedCount( 0 );
        camelContext.start();
        ...
```
Still no luck. `Rapidpro.flow.uuids` is still empty. Had the same issue in the first iteration of the connector when trying to inject the program stage codes. Ended up having to use `setProperty("programStageId", simple("{{program.stage.id}}")` in the route.

Reply:
Think I found something here:
https://stackoverflow.com/questions/17353327/populating-spring-value-during-unit-test
Will let you know how it goes.